Microsoft.HealthVault/Connection/HealthVaultConnectionBase.cs

```diff
@@ -178,6 +178,11 @@ protected HealthVaultConnectionBase(IServiceLocator serviceLocator)
             this.SessionCredential = await sessionCredentialClient.GetSessionCredentialAsync(token).ConfigureAwait(false);
         }
 
+        Task IConnectionInternal.RefreshSessionCredentialAsync(CancellationToken token)
```

Comment:
This is an explicit interface implementation, it can't have an access modifier.

---
In reply to: [115837978](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115837978) [](ancestors = 115837978)